libs/collectives/tests/performance/osu/CMakeLists.txt

```
@@ -0,0 +1,53 @@
+# Copyright (c) 2011 Bryce Adelstein-Lelbach
+# Copyright (c) 2012 Daniel Kogler
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+set(benchmarks
+    osu_bibw
+    osu_bw
+    osu_latency
+    osu_multi_lat)
+
+foreach(benchmark ${benchmarks})
+
+  set(sources
+      osu_base.cpp
+      ${benchmark}.cpp)
+
+  source_group("Source Files" FILES ${sources})
+
+  # add example executable
+  add_hpx_executable(${benchmark}
```

```suggestion
  add_hpx_executable(${benchmark}_test
```
The `add_hpx_performance_test` function expects the `_test` suffix... Same below for `coll_benchmarks`.